app/services/loader.js

```diff
@@ -36,7 +36,7 @@ export default Service.extend({
           resolve(data);
         },
         jqXHR => {
-          reject(`Could not make ${options.type} request to ${options.url}`);
+          reject(jqXHR.responseJSON ? jqXHR.responseJSON : `Could not make ${options.type} request to ${options.url}`);
```

Comment:
If json response is there, send that, 
else
If plain response is there, send that,
else
Send the default `Could not make ${options.type} request to ${options.url}`